### PR TITLE
update wayback.py

### PR DESCRIPTION
**Pull request by kimgwisoo**
line create

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ certs/
 savedfiles/
 settings.ini
 .env
+*.pyc
```
Comment (gnh1201) [inline .gitignore:5]: Please explanation

File: plugins/wayback.py [hunk 2]
```diff
@@ -10,58 +10,61 @@
 # Updated at: 2024-03-13
 #
 
+# 필요한 라이브러리와 모듈을 임포트합니다.
 import requests
-
 from server import Extension
+from decouple import config
```
Comment (gnh1201) [inline plugins/wayback.py:16]: Please explanation

**Review by gnh1201 — COMMENTED** [review]
Please check my review

File: plugins/wayback.py (hunk 2 above)
Comment (kimgwisoo) [reply plugins/wayback.py:16]: The 'wayback.py' file was missing a line to load the 'decouple' module, so I added it.

File: .gitignore (hunk 1 above)
Comment (kimgwisoo) [reply .gitignore:5]: Added the '*.pyc' extension to gitignore.

**Comment by gnh1201** [comment]
Thank you for the commit :)

**Pull request merged**